app/service/usda_food_service.rb

```diff
@@ -0,0 +1,7 @@
+class UsdaFoodService
+  def self.search(q)
+    query_params = {query: q, api_key: ENV['USDA_FOOD_DATA_KEY']}
+    res = Faraday.get('https://api.nal.usda.gov/fdc/v1/foods/search', query_params)
```

Comment:
I would set these query params in a block. That syntax is more readable/maintainable.